cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -498,7 +502,32 @@ var __AdyenHelper : Object = {
                 break;
 		}
 		return format;
-	}
+	},
+
+	getApplicationInfo : function () {
+
+        var externalPlatformVersion = "SiteGenesis";
```

Comment:
does this track pipeline integrations as well?

Reply:
yes, everything non-SFRA at the moment

Reply:
changed this, it now tracks only controllers/SFRA